### PR TITLE
chore: Postgres support

### PR DESCRIPTION
**Pull request by soluchok**
Signed-off-by: Andrii Soluk <isoluchok@gmail.com>

### CODE REVIEW

File: vct/Makefile [hunk 1]
```diff
@@ -27,6 +27,9 @@ export TRILLIAN1_LOG_MYSQL		?= orb1vct:orb1vct-secret-pw@tcp(mysql:3306)/orb1vct
 export TRILLIAN2_LOG_MYSQL		?= orb2vct:orb2vct-secret-pw@tcp(mysql:3306)/orb2vct
 export TRILLIAN3_LOG_MYSQL		?= orb3vct:orb3vct-secret-pw@tcp(mysql:3306)/orb3vct
```
Comment (fqutishat) [inline vct/Makefile:28]: need to remove trillian log mysql

File: vct/kustomize/vct/overlays/local/vct3/sedb64transform.yml [hunk 2]
```diff
@@ -8,4 +8,4 @@ apiVersion: svceng
 kind: SedB64Transformer
 metadata:
   name: sedb64transformer
-argsOneLiner: s^||COUCHDB_URL||^${COUCHDB_URL}^ s^||TRILLIAN3_LOG_MYSQL||^${TRILLIAN3_LOG_MYSQL}^
+argsOneLiner: s^||COUCHDB_URL||^${COUCHDB_URL}^ s^||TRILLIAN3_LOG_MYSQL||^${TRILLIAN3_LOG_MYSQL}^ s^||TRILLIAN3_LOG_POSTGRES||^${TRILLIAN3_LOG_POSTGRES}^
```
Comment (fqutishat) [inline vct/kustomize/vct/overlays/local/vct3/sedb64transform.yml:11]: need to remove TRILLIAN3_LOG_MYSQL

**Review by fqutishat — CHANGES_REQUESTED** [review]

**Comment by fqutishat** [comment]
@soluchok need to remove this https://github.com/trustbloc/k8s/blob/main/dbs/kustomize/dbs/components/mysql/configure_mysql.sql#L64

File: vct/kustomize/vct/overlays/local/vct3/sedb64transform.yml (hunk 2 above)
Comment (soluchok) [reply vct/kustomize/vct/overlays/local/vct3/sedb64transform.yml:11]: @fqutishat I did not remove it on purpose. What if we want to switch back for some reason?

**Review by fqutishat — APPROVED** [review]

**Pull request merged**